src/app/queries/fetchHelpers.ts

```diff
@@ -32,7 +33,7 @@ export const authorizedFetch = async <T>(
         ...extraHeaders,
       },
     });
-    if (response.ok && response.json) {
+    if (!isBlob && response.ok && response.json) {
```

Comment:
In this case if `isBlob` is true we'll always `throw`, right? don't we want to `return` in either case? Maybe something like this?

```
const responseBodyFn = !isBlob ? response.json : response.blob;
if (response.ok && responseBodyFn) {
  return responseBodyFn();
} else {
  throw response;
}
```

Reply:
Yeah, I discovered that afterwards, I was adding another condition but your approach is better. Thanks
I've changed it.